mokapot/plugins/__init__.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import sys
+import logging
+
+if sys.version_info >= (3, 10):
+    from importlib.metadata import entry_points
+else:
+    from importlib_metadata import entry_points
```

Comment:
**todo:** Change this to a `Try:... Except:...` block. I don't foresee this being a problem, but the try except solution should be a little more robust.

Reply:
done